quality/fca/cypress/integration/usager/connexion-fs-non-signé.feature

```diff
@@ -0,0 +1,9 @@
+#language: fr
+Fonctionnalité: Connexion Usager avec FS (user info non signé)
+  Scénario: Connexion OK 
+    Etant donné que je navigue sur la page fournisseur de service "sans signature de la réponse userinfo"
+    Et que je clique sur le bouton ProConnect
+    Et j'entre l'email "test@fia1.fr"
+    Et je clique sur le bouton de connexion
+    Alors je m'authentifie
+    Et je suis redirigé vers la page fournisseur de service "sans signature de la réponse userinfo"
```

Comment:
suggestion: this test actually show the standard way to connect : no signature required. I'd rather have a connexion-fs-signé.feature file test than this one. At this point, I suggest you add line 9 to connexion-fi.feature and delete this file.